pkg/api/validation/validation.go

```diff
@@ -1481,7 +1483,15 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 			allErrs = append(allErrs, field.Invalid(specPath.Child("storageClassName"), pv.Spec.StorageClassName, msg))
 		}
 	}
-
+	if pv.Spec.VolumeMode != nil && *pv.Spec.VolumeMode == api.PersistentVolumeBlock && !utilfeature.DefaultFeatureGate.Enabled(features.BlockVolumeSupport) {
```

Comment:
When the feature is disabled then **no** value of this field is valid, even "Filesystem"

Reply:
ack